### PR TITLE
Rename variables RHODS to RHOAI

### PR DESCRIPTION
**Pull request by ajaypratap003**
## Description

Rename variables RHODS to RHOAI

https://issues.redhat.com/browse/RHOAIENG-9642

## Merge criteria

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

Note: Don't merge this PR until upstream is not merge https://github.com/opendatahub-io/opendatahub-operator/pull/1277

### CODE REVIEW

**Comment by VaishnaviHire** [comment]
I think this PR needs to go upstream first

**Comment by ajaypratap003** [comment]
> I think this PR needs to go upstream first

@VaishnaviHire upstream PR is ready to review , please have look https://github.com/opendatahub-io/opendatahub-operator/pull/1277

File: pkg/cluster/cluster_config.go [hunk 1]
```diff
@@ -104,22 +104,22 @@ func detectSelfManaged(ctx context.Context, cli client.Client) (Platform, error)
 	return Unknown, nil
 }
 
-// detectManagedRHODS checks if catsrc CR add-on exists ManagedRhods.
-func detectManagedRHODS(ctx context.Context, cli client.Client) (Platform, error) {
+// detectManagedRHoAI checks if catsrc CR add-on exists ManagedRhoai.
```
Comment (zdtsw) [inline pkg/cluster/cluster_config.go:107]:
```suggestion
// detectManagedRHOAI checks if catsrc CR add-on exists ManagedRhoai.
```

**Review by zdtsw — COMMENTED** [review]
why not do the change in ODH first, otherwise i am fine with the change

**Comment by VaishnaviHire** [comment]
Closing this, as we will cherry-pick the upstream PR

**Pull request closed without merging**